data/windows.yaml

```diff
@@ -109,6 +109,60 @@ urls:
 - 'http://www.hexacorn.com/blog/2013/01/19/beyond-good-ol-run-key-part-3/'
 - 'https://msdn.microsoft.com/en-us/library/windows/desktop/ee872121(v=vs.85).aspx'
 ---
+name: WindowsApplicationShimFiles
+doc: |
+  Windows Application Shim Files
+
+  Malicious shim database files can be used to bypass UAC, inject DLLs into
+  legitimate processes, and modify anti-virus software to avoid detection.
+
+  drvmain.sdb, frxmain.sdb, msimain.sdb, pcamain.sdb, and sysmain.sdb are
+  shim database files that ship with Windows, and contain many predefined
+  shims that address known program incompabilities. These files are not
+  signed, so malware can insert shim entries into them.
+
+  The other paths point to the locations where SDB files are installed
+  when the sdbinst.exe utility is used to install them.
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '%%environ_windir%%\AppPatch\drvmain.sdb'
+    - '%%environ_windir%%\AppPatch\frxmain.sdb'
+    - '%%environ_windir%%\AppPatch\msimain.sdb'
+    - '%%environ_windir%%\AppPatch\pcamain.sdb'
+    - '%%environ_windir%%\AppPatch\sysmain.sdb'
+    - '%%environ_windir%%\AppPatch\AppPatch64\Custom\*'
+    - '%%environ_windir%%\AppPatch\Custom\*'
+    - '%%environ_windir%%\AppPatch\Custom\Custom64\*'
+    - '%%environ_windir%%\AppPatch\CustomSDB\*'
+    separator: '\'
+labels: [Users]
+supported_os: [Windows]
+urls:
+- 'https://attack.mitre.org/techniques/T1138/'
+- 'https://countercept.com/blog/hunting-for-application-shim-databases/'
+- 'http://files.brucon.org/2015/Tomczak_and_Ballenthin_Shims_for_the_Win.pdf'
+- 'https://www.blackhat.com/docs/eu-15/materials/eu-15-Pierce-Defending-Against-Malicious-Application-Compatibility-Shims-wp.pdf'
+---
+name: WindowsApplicationShimRegistryKeys
+doc: |
+  Windows Application Shims Registry Keys
```

Comment:
Registry => Windows Registry

Reply:
and Keys => Values

Reply:
maybe rename this key to closer reflect its purpose? isn't it used to store the location of the shim database files?

Reply:
@joachimmetz yeah, InstalledSDB\*\DatabasePath stores the path to the SDB, and Custom\* stores the mapping between the application that the SDB identifier and the file name that it applies to.  How about renaming it to something like `WindowsApplicationShims'?  I originally added "RegistryKeys" to differentiate it from the artifact that pertained to the corresponding files on disk, but changing that artifact name to 'WindowsApplicationShimDatabaseFiles' solves that problem.

Reply:
Maybe split them in:
* WindowsApplicationCompatibilityInstalledShimDatabases
* WindowsApplicationCompatibilityShimDatabaseMappings

You can create additional artifact group if it is convenient to collect them at the same time

Reply:
Do you have any recommendations for the name of that artifact group?

Reply:
Maybe `WindowsApplicationCompatibilityShims`